bin/compile

```diff
@@ -28,17 +28,17 @@ trap 'err_trap' ERR
 
 # stdlib
 # download to a file first, as the function restarts the entire download on code 18 connection reset (not all servers support -C)
-curl_retry_on_18 --fail --silent --location -o $bp_dir/bin/util/stdlib.sh https://lang-common.s3.amazonaws.com/buildpack-stdlib/v8/stdlib.sh || {
+curl_retry_on_18 --fail --silent --location -o ${TMPDIR:-/tmp}/stdlib.sh https://lang-common.s3.amazonaws.com/buildpack-stdlib/v8/stdlib.sh || {
 	error <<-EOF
 		Failed to download standard library for bootstrapping!
-		
+
 		This is most likely a temporary internal error. If the problem
 		persists, make sure that you are not running a custom or forked
 		version of the Heroku PHP buildpack which may need updating.
 	EOF
 }
-source $bp_dir/bin/util/stdlib.sh
-rm $bp_dir/bin/util/stdlib.sh
+source ${TMPDIR:-/tmp}/bin/util/stdlib.sh
+rm ${TMPDIR:-/tmp}/bin/util/stdlib.sh
```

Comment:
Hi @jkutner,

i think you missed to remove the `/bin/util` path of the path

Reply:
thanks